geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/StringBytesGlossary.java

```diff
@@ -32,27 +32,37 @@
   /**
    * byte identifier of a bulk string
    */
-  public static final byte BULK_STRING_ID = 36; // '$'
+  public static final byte BULK_STRING_ID = '$';
```

Comment:
Is a java character literal's ordinal value stable? Or can it change based on the locale? If you know the answer can you send a link?

Reply:
The only link I have is the java language spec - https://docs.oracle.com/javase/specs/jls/se16/html/jls-3.html#jls-3.10.4. The character literals are UTF-16.